pkg/storage/storage.go

```diff
@@ -68,3 +71,18 @@ func (s *storage) Store(batch *MetricsBatch) {
 	s.nodes.Store(batch.Nodes)
 	s.pods.Store(batch.Pods)
 }
+
+// uint64Quantity converts a uint64 into a Quantity, which only has constructors
+// that work with int64 (except for parse, which requires costly round-trips to string).
+// We lose precision until we fit in an int64 if greater than the max int64 value.
+func uint64Quantity(val uint64, format resource.Format, scale resource.Scale) resource.Quantity {
```

Comment:
would it make sense to move this function to storage/types.go as it is only used to get the resource usage?

Reply:
As uint64Quantity is called only from `storage/types.go` it makes sense.